editor/editor_node.cpp

```diff
@@ -4824,9 +4824,21 @@ void EditorNode::_scene_tab_changed(int p_tab) {
 	editor_data.get_undo_redo().commit_action();
 }
 
+String EditorNode::capitalize_string(const String &p_string) const {
```

Comment:
We should avoid putting all kinds of helpers in EditorNode as it makes it *huge*. Not sure where to put this though, maybe @EricEzaM has an idea since he's been reworking this code.

Reply:
Looks like it can go into `String`.

Reply:
No, in Godot **solutions must be local** and such editor-specific helper code is not something that should be part of the core String API (just like we wouldn't try to add it to `std::string` if we used the STL).

https://docs.godotengine.org/en/latest/community/contributing/best_practices_for_engine_contributors.html#solutions-must-be-local